src/eval.rs

```diff
@@ -82,11 +83,7 @@ pub enum ValRef {
 
 impl ValRef {
     pub fn to_bool(&self) -> bool {
-        match self {
-            ValRef::Bool(false) => false,
-            ValRef::None => false,
-            _ => true,
-        }
+        !matches!(self, ValRef::Bool(false) | ValRef::None)
```

Comment:
This is shorter, but I wonder if it's better? The old code was very easy to read: "if the value is the boolean `false`, it's false, if the value is `none` it's false, otherwise it's true". The new code reads like, "the opposite of whether the value matches the bool false or the value none", which seems less clear to me.

Reply:
Hm, clippy told me to do it like that, and it is indeed shorter. From the reading perspective it seems pretty similar to me.